tests/modules.ownableValidator.write.test.ts

```diff
@@ -0,0 +1,275 @@
+import {
+  http,
+  type Account,
+  type Chain,
+  type Hex,
+  type WalletClient,
+  createWalletClient,
+  parseEther,
+  zeroAddress,
+  encodePacked
+} from "viem"
+import { afterAll, beforeAll, describe, expect, test } from "vitest"
+import {
+  type NexusSmartAccount,
+  createSmartAccountClient
+} from "../src/account"
+import { type TestFileNetworkType, toNetwork } from "./src/testSetup"
+import {
+  getTestAccount,
+  killNetwork,
+  toTestClient,
+} from "./src/testUtils"
+import type { MasterClient, NetworkConfig } from "./src/testUtils"
+import { createK1ValidatorModule, createOwnableValidatorModule, getRandomSigner } from "../src/modules/index"
+import { OWNABLE_VALIDATOR_ADDRESS } from "../src/account/utils/Constants"
+import { OwnableValidator } from "../src/modules/validators/OwnableValidator"
+import { K1ValidatorModule } from "../src/modules/validators/K1ValidatorModule"
+
+const NETWORK_TYPE: TestFileNetworkType = "PUBLIC_TESTNET"
```

Comment:
why resort to public testnet? any particular reason?

Reply:
fixed